bin/dump_fedora_pids.rb

```diff
@@ -0,0 +1,46 @@
+## assumptions:
+# * you're running this on the same machine as the fedora instance you want to dump from
+# * the user you're running it as can execute the 'mysql' command, and can write to '/tmp'
+# * the user can temporarily write a sql script to the place from which their running this script
+# * .fedora.my.cnf is a config file with a line like "  password = supersecretcredential"
+# * the password in .fedora.my.cnf will get you into fedora sql db for it's user (which defaults to "fedora")
+# * .fedora.my.cnf exists in the same directory from which you're running this script, and can be read by the user running it
+#
+# if all that's true, you should be able to do "ruby dump_fedora_pids.rb"
+# and it should tell you where it wrote the pids
+
+require 'date'
+
+cur_datetime_str = DateTime.now.strftime('%Y-%m-%d_%H:%M:%S')
+fedora_conf_file_name = '.fedora.my.cnf'
+out_file_name = "/tmp/all_pids_#{cur_datetime_str}"
+script_file_name = "fedora_mysql_pid_dump_#{cur_datetime_str}.sql"
+
+
+def get_fedora_password conf_file_name
+  fedora_conf_file_contents = open(conf_file_name, 'r').read
+  return (/^\s*password\s*=\s*(.*)$/.match(fedora_conf_file_contents))[1]
```

Comment:
Instead of just removing the parens, maybe this is a good time to refactor that method further?

It seems like this line could be split into multiple, highly readable lines (and might benefit from it -- it seems like some error handling if the was no match might be useful).